cmd/k8s-operator/operator_test.go

```diff
@@ -42,11 +47,13 @@ func TestLoadBalancerClass(t *testing.T) {
 			operatorNamespace: "operator-ns",
 			proxyImage:        "tailscale/tailscale",
 		},
-		logger: zl.Sugar(),
+		logger:   zl.Sugar(),
+		clock:    clock,
+		recorder: record.NewFakeRecorder(100),
 	}
 
-	// Create a service that we should manage, and check that the initial round
-	// of objects looks right.
+	// Create a service that we should manage, but start with a miconfiguration
```

Comment:
I was in two minds about updating the existing test or creating a new test for the error conditions - happy to split the error conditions out if you think the test is becoming too unwieldy.

Reply:
I really like the bit that shows that when a cleanup happens the conditions get removed- I think it makes sense to have the tests combined as you did 👍🏼